drafts/GoL-reules/Game of Links.md

```diff
@@ -0,0 +1,139 @@
+# The Game of Links. Goals, rules and provision.
+
+## Intro
+
+
+## General provision of the Game.
+
+![](euler-5_timeline.png)
+
+We want to reproduce the whole process of mainnet launch in `Euler-5` for testing auction, drop and distribution models. As shown in figure above the Game isn't start with `Euler-5` launch. We want to allow some time to participants of the Game for their funds checking and hardware and software setuping. Also, in that 15-days limb the Gamers can decide on what exactly discipline they want to participate and strategize.
+
+`Euler-5` Genesis file will contain all drops to Ethereum and Cosmos communities. Furthermore,  it will contain validators lifetime rewards from `Euler-4`. For new incomers, we'll propose to get CYB tokens at the Auction just for less. In the Auction, newbies can buy CYBs or GOLs for uAtoms and Szabos. 
+
+> ***Few words about the Auction.***
+
+With the Game of Links start after 15 days of `Euler-5` launch, we will start tracking the progress at the Game monitor and store data. Then, and only then. The monitor will be available at the Brain block explorer by cyberd.ai address. 
```

Comment:
I think that cyberd.ai changes must be optional. I dont sure that we will be able to deliver this ahead. Probably during...